src/core/messageSender.ts

```diff
@@ -0,0 +1,42 @@
+import * as TelegramBot from 'node-telegram-bot-api';
+import { Message } from 'node-telegram-bot-api';
+
+import TextResponse from '../classes/TextResponse';
+import LocationResponse from '../classes/LocationResponse';
+import ImageResponse from '../classes/ImageResponse';
+import VoiceResponse from '../classes/VoiceResponse';
+import UseCaseResponse from '../classes/UseCaseResponse';
+import TextToSpeechConnector from '../connectors/textToSpeech/textToSpeechConnector';
+
+export default class MessageSender {
+  private textToSpeech = new TextToSpeechConnector();
+
+  private chatId: number;
```

Comment:
Angedacht war es diese Daten statisch zu halten im BrainState  und nicht bei jedem aufruf zu setzen, da wir mit nur einem User rechnen.

Reply:
Wir können die Chat ID auch nur beim ersten Mal setzen, aber da die Chat ID eh nur im MessageSender verwendet wird, würde ich das in der Klasse lassen.